datajunction-ui/src/app/pages/AddEditNodePage/index.jsx

```diff
@@ -281,6 +311,54 @@ export function AddEditNodePage() {
                   </div>
                 );
 
+                const metricMetadataInput = (
+                  <>
+                    <div
+                      className="MetricKindInput NodeCreationInput"
+                      style={{ width: '25%' }}
+                    >
+                      <ErrorMessage name="metric_kind" component="span" />
+                      <label htmlFor="MetricKind">Metric Kind</label>
+                      <Field as="select" name="metric_kind" id="MetricKind">
+                        {metricKinds.map(kind => (
+                          <option value={kind}>{labelize(kind)}</option>
+                        ))}
+                      </Field>
+                    </div>
+
+                    <div
+                      className="MetricDirectionInput NodeCreationInput"
+                      style={{ width: '25%' }}
+                    >
+                      <ErrorMessage name="metric_direction" component="span" />
+                      <label htmlFor="MetricDirection">Metric Direction</label>
+                      <Field
+                        as="select"
+                        name="metric_direction"
+                        id="MetricDirection"
+                      >
+                        {metricDirections.map(direction => (
+                          <option value={direction}>
```

Comment:
Should we explicitly catch the default values of "Neutral", Unknown", and "Unspecified" and then set a placeholder value instead like "Set metric direction..."? I think that might be more inviting for users to go ahead and set a value even though it's optional. On that same note, can we show an asterisk on the required fields so that it's clear these are optional?

Reply:
Makes sense, I added a blank option to each of these, so that it clearly indicates it's optional. Also added asterisks to the required fields